ui/components/identity-item.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react'
+
+import DeleteModal from './delete-modal'
+
+export default function IdentityItem({
+  item,
+  deleteModalInfo,
```

Comment:
Two parameters would be better here to reduce coupling

Reply:
which two parameters? O.o